code/style.css

```diff
@@ -1,4 +1,188 @@
+@import url('https://fonts.googleapis.com/css2?family=Oswald:wght@600&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=EB+Garamond:wght@400;500&display=swap');
+
+
 html {
-  background: #0025ff;
-  color: #fff;
+background: #FFF; 
 }
+
+body {
+	margin: 0;
+}
+
+	.container {
+		width: 320px;
```

Comment:
Just a heads up that this `width` will limit the width of all content:
<img width="659" alt="Skärmavbild 2021-02-03 kl  15 03 22" src="https://user-images.githubusercontent.com/51529201/106757691-f6cc9c80-6630-11eb-9b4f-2ed9deadcb33.png">